modules/monitoring-managing-core-platform-alerting-rules.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * monitoring/managing-alerts.adoc
+
+:_content-type: CONCEPT
+[id="managing-core-platform-alerting-rules_{context}"]
+= Managing alerting rules for core platform monitoring
+
+:FeatureName: User-defined and user-modified alerting rules for core platform monitoring
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+{product-title} {product-version} monitoring ships with a large set of default alerting rules for platform metrics.
+As a cluster administrator, you can customize this set of rules in two ways:
+
+* Modify the settings for existing platform alerting rules by adjusting thresholds or by adding and modifying labels.
```

Comment:
note: only AlertingRule resources created in the `openshift-monitoring` namespace will be reconciled by CMO